libs/feature/catalog/src/lib/feature-catalog.module.ts

```diff
@@ -26,5 +31,29 @@ import { UiElementsModule } from '@geonetwork-ui/ui/elements'
     UiElementsModule,
   ],
   exports: [SiteTitleComponent, SourceLabelComponent, OrganisationsComponent],
+  providers: [
+    {
+      provide: OrganisationsServiceInterface,
+      useClass: OrganisationsFromMetadataService,
+    },
+  ],
 })
-export class FeatureCatalogModule {}
+export class FeatureCatalogModule {
+  static withOrganizationsStrategy(
```

Comment:
maybe would need some documentation on how to change the strategy within your app ?

Reply:
Yes definitely! I will recreate this PR on upstream because nothing will be specific to geocat